Source/CarthageKit/Xcode.swift

```diff
@@ -254,7 +254,7 @@ public func schemesInProject(project: ProjectLocator) -> SignalProducer<String,
 		}
 		.skipWhile { line in !line.hasSuffix("Schemes:") }
 		.skip(1)
-		.takeWhile { line in !line.isEmpty }
+		.takeWhile { line in !line.isEmpty && line.hasPrefix("  ") }
```

Comment:
This seems very fragile. Xcode could change the prefix format at any time and this would break :/